app/bundles/LeadBundle/Controller/CompanyController.php

```diff
@@ -209,7 +209,7 @@ public function newAction(Request $request, $entity = null)
         $leadFieldModel = $this->getModel('lead.field');
         \assert($leadFieldModel instanceof FieldModel);
         $fields = $leadFieldModel->getPublishedFieldArrays('company');
-        $form   = $model->createForm($entity, $this->formFactory, $action, ['fields' => $fields, 'update_select' => $updateSelect]);
+        $form   = $model->createForm($entity, $this->formFactory, $action, ['fields' => $fields, 'update_select' => $updateSelect, 'hide_extra_buttons' => true]);
```

Comment:
1. Let's call it what it really does.
2. Let's avoid double negatives as those are hard to comprehend and break my brain (like hide = false) 
```suggestion
        $form   = $model->createForm($entity, $this->formFactory, $action, ['fields' => $fields, 'update_select' => $updateSelect, 'show_merge_button' => false]);
```
Or, perhaps we could avoid this extra config option altogether if we'd base the condition if the company ID exists or not?

Reply:
Yes! I'll do the change